helm/examples/azure/sa/README.md

```diff
@@ -0,0 +1,74 @@
+# HPCC Azure storage with a Storage Account
+
+## Directory contents
+
+### Introduction
+Persistant storage can be created manually or through Kubernetes Cluster. If created through Kubernetes the storage will be gone when Perstiant Volume (PV) or Kubernetes cluster are deleted. For Azure manually create a shared storage with storage account will not depends on any Kubernetes cluster. Also the data generated will be persistent and be re-used as long the PV/PVC names are the same.
+
+
+### Create a shared with a storage account
+
+To create a shared storage including storage account you can reference Reference: https://docs.microsoft.com/en-us/azure/aks/azure-files-volume
+We also provide a set of scripts to help the processes. But before you run any script make sure login to Azure first:
+```console
+az login
+```
+
+### Environment variables
+env-sa define some variables for creating storage account and shared storage.
+Some variables have default values but some of them don't and you must provide the values:
+- SUBSCRIPTION: Azure subscription
+- STORAGE_ACCOUNT_NAME: Storage Account name which will be generated
+- SA_REOUSRCE_GROUP: a resource group name which must be unique in your subscription
+- SA_LOCATION: the region of storage account will be created. It is recommended to use the same region as the Kubernetes cluster region.
+
+hpcc-azurefile/values.yaml defines tree variables needed for storage account and shared storage creation:
+- secretName:  a Kuberentes secret
+- secretNamespace: the namespace of the Kubernetes secret
+- shareName: shared storage name which will be used to create the storage
+
+### Create
+create-sa.sh will source env-sa, create a storage account and shared storage. It will also save the storage key to $SA_KEY_DIR/
+```console
+./create-sa.sh
+```
+### List
+list-sa.sh will list the shared storage
+```console
+./list-sa.sh
+```
+
+
+### Create a Kubernetes secret
+A kubernetes secrete is required to use the storage.  The secret name, secreteName,is defined in hpcc-azurefile/values.yaml.<br/>
+If this Kuberenetes cluster has never generated the secret before, run following to generate the secret
+```console
+./create-secret.sh
+```
+### Install and use hpcc-azurefile
+
+The hpcc-azurefile helm chart will create Persistent Volume and Persistent Volume Claim for each of the required HPCC storage types based on the shared storage
```

Comment:
The hpcc-azurefile helm chart will create the Persistent Volume and the Persistent Volume Claim for each of the required HPCC storage types based on the shared storage.

Reply:
Fixed